src/main/java/com/rackspace/jenkins_nodepool/NodePool.java

```diff
@@ -231,6 +235,16 @@ public NodePool(String connectionString,
         return acceptedNodes;
     }
 
+    public String getRequestTimeout() {
+        return requestTimeout;
+    }
+
+    public final void setRequestTimeout(String requestTimeout) {
+        // ensure value can be parsed as an integer
+        Integer.parseInt(requestTimeout);
```

Comment:
May want to ensure value is a positive value > 0.  After parsing, `if Integer.parseInt(requestTimeout) < 1 throw new IllegalArgumentException("Timeout value must be positive.");` A zero or negative value is undesirable.  Maybe add javadoc to indicate the value is in seconds??  I see we have form validation below to ensure that it's positive, but we can also add checks on the implementation.  We can easily add a quick unit test to the method for this too...